src/utils/getHyperLinkFromPath/getHyperLinkFromPath.ts

```diff
@@ -1,19 +1,19 @@
 /** generates the clickable link to a specific folder
  * @param {string} path - The path of a specific folder like "src/components/reusable"
  * @returns {string} - the hyperLink like "[reusable](./src/components/reusable)"
-*/
+ */
 
 export const getHyperLinkFromPath = (path: string): string => {
-  const pattern = /^((?![<>:"/\\|?* ])(([a-z0-9\s_@\-^!#$%&+={}\\[\].]*)([/]?)))+[^/.]$/i;
-  if (pattern.test(path)) {
+  const PATTERN = /^((?![<>:"/\\|?* ])(([a-z0-9\s_@\-^!#$%&+={}\\[\].]*)([/]?)))+[^/.]$/i;
```

Comment:
What does this PATTERN mean? 
Is this PATTERN supposed to be (FOLDER_PATTERN???)